### DISCUSSION TITLE
short form link change

### DISCUSSION THREAD

**Discussion by loskotek** [pretalx/pretalx#1390] [Self-hosting] [answered]
Hey, in general settings, I am unable to change the short form despite being an administrator of the event. Anyone experiencing the same issue? Any ideas how to solve it?

Thanks.

**Comment by rixx** [comment, marked as answer]
Administrators used to have to go to the shell or database to change an event's name.

However, I just looked at the code and since the fix for this seems to be completely trivial, I just pushed a change in 6af4db5e that should fix the issue, and allow administrators to use the frontend to change event short forms/slugs.
